src/image.cpp

```diff
@@ -307,166 +307,167 @@ namespace librealsense
         assert(n % 16 == 0); // All currently supported color resolutions are multiples of 16 pixels. Could easily extend support to other resolutions by copying final n<16 pixels into a zero-padded buffer and recursively calling self for final iteration.
 #ifdef RS2_USE_CUDA
         rscuda::unpack_yuy2_cuda<FORMAT>(d, s, n);
-                  
-
-#elif __SSSE3__
-        static bool do_avx = has_avx();
-
-        if (do_avx)
-        {
-            if (FORMAT == RS2_FORMAT_Y8) unpack_yuy2_avx_y8(d, s, n);
-            if (FORMAT == RS2_FORMAT_Y16) unpack_yuy2_avx_y16(d, s, n);
-            if (FORMAT == RS2_FORMAT_RGB8) unpack_yuy2_avx_rgb8(d, s, n);
-            if (FORMAT == RS2_FORMAT_RGBA8) unpack_yuy2_avx_rgba8(d, s, n);
-            if (FORMAT == RS2_FORMAT_BGR8) unpack_yuy2_avx_bgr8(d, s, n);
-            if (FORMAT == RS2_FORMAT_BGRA8) unpack_yuy2_avx_bgra8(d, s, n);
-        }
-        else
-        {
-            auto src = reinterpret_cast<const __m128i *>(s);
-            auto dst = reinterpret_cast<__m128i *>(d[0]);
+#endif
+#ifndef ANDROID
+    #ifdef __SSSE3__
```

Comment:
No need for extra indentation after `#ifndef ANDROID`, its better to keep the diff short